test/unit/org/apache/cassandra/db/compaction/CompactionsCQLTest.java

```diff
@@ -245,6 +256,174 @@ public void testPerCFSNeverPurgeTombstones() throws Throwable
         testPerCFSNeverPurgeTombstonesHelper(false);
     }
 
+    @Test
+    public void testCompactionInvalidRTs() throws Throwable
+    {
+        DatabaseDescriptor.setCorruptedTombstoneStrategy(Config.CorruptedTombstoneStrategy.exception);
+        prepare();
+        RangeTombstone rt = new RangeTombstone(Slice.ALL, new DeletionTime(System.currentTimeMillis(), -1));
```

Comment:
For each test case can you explain exactly what field you are setting that is invalid and then ELI5 reason why that value isn't valid.